playwright-e2e/fixtures/fixture-base.ts

```diff
@@ -18,6 +19,12 @@ export const fixtureBase = base.extend({
     await page.route('**/*', async (route) => {
       return REQUEST_EXCLUDES.some((urlPart) => route.request().url().includes(urlPart)) ? route.abort() : route.continue();
     });
+    // Listen for all console events and log errors to the Terminal window
+    page.on('console', msg => {
+      if (msg.type() === 'error') {
+        logInfo(`Browser Console Error: "${msg.text()}"`);
+      }
+    });
```

Comment:
Write browser console error to test log.

Example

<img width="694" alt="Screenshot 2024-01-30 at 2 17 45 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/5c086a5a-8b98-409d-8023-bd405503e8b5">